galette/templates/default/pages/configuration_dynamic_fields.html.twig

```diff
@@ -28,6 +28,14 @@
 
 {% block javascripts %}
         <script type="text/javascript">
+            $('.menu.tabbed').tab({
```

Comment:
Both solutions (path+auto or apisettings) have the same behavior:
- current tab is called from ajax
- no other ajax call is done when switching tabs :/

The call to currently loaded tab should not be done, I've not been able to prevent that (I though using `data-loaded=""` should make the work but it does not)

Reply:
This is because tabs are already initialized globally in `common.js` on every menu using the `tabbed` class, and because tabs should be initialized on the activating items.

First solution : remove the `tabbed` class on the menu on line 4 and change the selector.
```suggestion
            $('.pointing.menu .item').tab({
```

Reply:
:+1: